packages/gen-ai-hub/src/index.ts

```diff
@@ -1,10 +1,80 @@
-export * from './client/index.js';
+export type {
+  OpenAiChatModel,
+  OpenAiEmbeddingModel,
+  OpenAiChatMessage,
+  OpenAiChatSystemMessage,
+  OpenAiChatUserMessage,
+  OpenAiChatAssistantMessage,
+  OpenAiChatToolMessage,
+  OpenAiChatFunctionMessage,
+  OpenAiChatCompletionFunction,
+  OpenAiChatCompletionTool,
+  OpenAiChatFunctionCall,
+  OpenAiChatToolCall,
+  OpenAiCompletionParameters,
+  OpenAiChatCompletionParameters,
+  OpenAiEmbeddingParameters,
+  OpenAiCompletionOutput,
+  OpenAiChatCompletionOutput,
+  OpenAiPromptFilterResult,
+  OpenAiContentFilterResultsBase,
+  OpenAiContentFilterPromptResults,
+  OpenAiContentFilterResultBase,
+  OpenAiContentFilterDetectedResult,
+  OpenAiContentFilterSeverityResult,
+  OpenAiEmbeddingOutput
+} from './client/index.js';
+export { OpenAiClient } from './client/index.js';
+
+export type {
+  ModelDeployment,
+  DeploymentIdConfiguration,
+  FoundationModel,
+  ModelConfiguration
+} from './utils/index.js';
 export {
-  OrchestrationClient,
+  isDeploymentIdConfiguration,
+  getDeploymentId,
+  resolveDeployment
+} from './utils/index.js';
+
+export type {
   OrchestrationCompletionParameters,
   CompletionPostResponse,
-  azureContentFilter,
   PromptConfig,
   LlmConfig,
-  ChatMessages
+  ChatMessages,
+  UnmaskingConfig,
+  TokenUsage,
+  TemplatingModuleConfig,
+  TemplatingModuleResult,
+  PresidioEntities,
+  OrchestrationConfig,
+  ModuleResults,
+  ModuleConfigs,
+  Masking,
```

Comment:
Some of these do not seem to exist 🤔

Reply:
Ah i guess some of them got updated/deleted after the new orchestration spec was used to generate clients.